src/components/Person/Person.jsx

```diff
@@ -1 +1,43 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const whicSex = sex === 'm'
+    ? `wife`
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {age && (<p className="Person__age">{`I am ${age}`}</p>)}
+
+      <p className="Person__partner">
+        {isMarried ? `${partnerName} is my ${whicSex}`
+          : 'I am not married'}
+      </p>
+    </section>
+  );
+};
+
+Person.defaultProps = {
+  age: null,
+  isMarried: null,
+  partnerName: null,
+};
+
+Person.propTypes = {
+  name: PropTypes.string.isRequired,
+  age: PropTypes.number,
+  sex: PropTypes.string.isRequired,
+  isMarried: PropTypes.bool,
+  partnerName: PropTypes.string,
+};
```

Comment:
Create the propType for the person object too. 
```suggestion
Person.propTypes = {
  person: PropTypes.shape({
    name: PropTypes.string.isRequired,
    age: PropTypes.number,
    sex: PropTypes.string.isRequired,
    isMarried: PropTypes.bool,
    partnerName: PropTypes.string,
  }).isRequired,
};
```

Reply:
dont matter what is inner structure of object in this case.  we need only parametrs and i did it. but I'll play with :)